### ISSUE TITLE
Make the "choose content type" menus consistent across Argo and preassembly

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/argo#2427]
We should use the same content type terminology for content types across all menus where content type can be set. This includes:

- Argo registration form
- Argo item show page
- Argo bulk update page
- preassembly (though this might not be possible as currently designed)

The same list of allowable content types should be presented in each list.

|content type|definition|
-----------------|------------------
|Image| One or more images|
| book (ltr) | Book (left-to-right reading order) |
| book (rtl) | Book (right-to-left reading order) |
| document | Only supports a single PDF file |
| file | Any file, will appear as a list of files |
| geo | Geographic data |
| map | For maps (treated same way as image) |
| media | Media objects (audiovisual-based content) |
| webarchive-seed | Web archive seed objects |
| 3d | 3d objects |

**Comment by jcoyne** [comment]
The reading direction is really not part of the content type.  It's a separate attribute that is only available when the content type is book.

**Comment by andrewjbtw** [comment]
Right. I can go either way on that one. Currently the registration form allows you to pick reading order but nothing actually happens when you do that. You just get left-to-right. We should either have options that work or not have those options.

If we have reading order in the menu, it's not technically a content type distinction but it could be used as a convenient way to set the order using existing menus. Without reading order in the menu, we either have to support a direct edit (currently how it's done) or a new design where reading order becomes selectable if the content type is a book.

**Comment by peetucket** [comment]
It looks like the reading direction makes a difference in the cocina-model when registering items in argo: https://github.com/sul-dlss/argo/blob/main/app/forms/registration_form.rb#L36-L40

Does that help decide whether it makes sense to leave that distinction in place and try to make sure it propogates correctly into the model from pre-assembly as well?

Also, for any extra content types in the list above that aren't currently in the forms, do we need new mappings into the cocina model?

**Comment by peetucket** [comment]
Argo also has 'software', not in the list above.  Remove it?

**Comment by peetucket** [comment]
I answered my second question ... it looks like the new content types not currently shown in argo are:

geo - has a mapping at https://github.com/sul-dlss/cocina-models/blob/main/lib/cocina/models/vocab.rb#L43-L45
webarchive-seed - has a mapping at https://github.com/sul-dlss/cocina-models/blob/main/lib/cocina/models/vocab.rb#L91-L93

Anything that doesn't map goes to `object`: https://github.com/sul-dlss/argo/blob/main/app/forms/registration_form.rb#L72-L73 but i think we will have all the options covered (if we remove `software` that is)

**Comment by andrewjbtw** [comment]
I think we can remove software. Nothing has that type and it's not listed as a defined type in the Chimera documentation.

**Comment by andrewjbtw** [comment]
@peetucket with book reading-order, I didn't realize Cocina was already writing the correct order for the menu selection. Reading order hasn't been written to contentMetadata, so users add it by editing the datastream. In that case, I'd like to leave the distinction in place.

**Comment by andrewjbtw** [comment]
Also, should I write a ticket for the corresponding work in preassembly? I'm realizing we probably should exclude geo from the preassembly menu because I think it involves a whole other server (kurma?).

**Comment by peetucket** [comment]
Ok, I'll leave the reading order options in argo and exclude Geo from pre-assembly.  I think a separate ticket for reading order in pre-assembly makes sense... I haven't had a chance yet to figure out to deal with that in pre-assembly. It may be straightforward, but not sure yet.  Maybe reference this ticket

**Comment by peetucket** [comment]
So I have a PR now in Argo (https://github.com/sul-dlss/argo/pull/2491), but when I went to pre-assembly (https://github.com/sul-dlss/pre-assembly/pull/758), I realized that pre-assembly generates `contentMetadata`, and two of the styles in the list that are missing in pre-assembly (`webarchive_seed` and `media`) are not supported in the `assembly-objectfile` gem that pre-assembly uses to make `contentMetadata`.  See https://github.com/sul-dlss/assembly-objectfile/blob/main/lib/assembly-objectfile/content_metadata.rb#L17
 
So I don't think adding those two styles will work right now in pre-assembly.

**Comment by andrewjbtw** [comment]
I filed a ticket for reading order in preassembly: https://github.com/sul-dlss/pre-assembly/issues/759

I already had a ticket for webarchive-seed in preassembly but didn't know about the assembly-objectfile issue https://github.com/sul-dlss/pre-assembly/issues/753 I'd hoped that since a webarchive-seed is just one image resource + the content type it would be simpler, but alas.

**Comment by peetucket** [comment]
Closed by sul-dlss/pre-assembly#758
